pkg/networkmanager/network-interface.jsx

```diff
@@ -180,6 +180,8 @@ export const NetworkInterfacePage = ({
                 desc = _("VLAN");
             } else if (dev.DeviceType == 'bridge') {
                 desc = _("Bridge");
+            } else if (dev.Driver == 'wireguard') {
+                desc = "WireGuard";
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17653-20220819-103252-ff5ac59f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)